### PR TITLE
Add Lewis

### PR DESCRIPTION
**Pull request by minijackson**
The Python based simulator framework from ESS: https://github.com/ess-dmsc/lewis

Also migrate the psu_simulator to Lewis, and use that as a server for testing StreamDevice.

Best viewed commit by commit.

### CODE REVIEW

File: ioc/tests/support/StreamDevice/simple/top/simpleApp/Db/simple.db [hunk 1]
```diff
@@ -1,81 +1,82 @@
...
-record(ai, "FLOAT:IN") {
-	field(DESC, "A fixed float")
+record(ao, "${P}UCmd") {
+	field(DESC, "Voltage command")
 	field(DTYP, "stream")
-	field(INP, "@simple.proto getFloat $(PORT)")
-	field(PINI, "YES")
-	field(SCAN, "5 second")
+	field(OUT, "@simple.proto setUCmd $(PORT)")
+	field(EGU, "A")
```
Comment (agaget) [inline ioc/tests/support/StreamDevice/simple/top/simpleApp/Db/simple.db:7]: A is for a measure of intensity not Voltage. Same for the other voltages measure of current

**Comment by agaget** [comment]
ok for me then.
I can't find any "approve" button !?

**Review by agaget — APPROVED** [review]
fine for me

**Review by stephane-cea — APPROVED** [review]
This looks good to me!

**Pull request merged**